recipes/snappy/meta.yaml

```diff
@@ -0,0 +1,25 @@
+{% set version = "1.1.3" %}
+
+package:
+  name: snappy
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/google/snappy.git
+  git_tag: {{ version }}
+
+requirements:
+  build:
+    - msinttypes  # [win]
+    - libtool # [unix]
+    - automake # [unix]
+
```

Comment:
With tests in make check, the tests here usually just test existence of the
libraries that should be created.

On Wed, Mar 30, 2016, 18:55 Ryan Grout <notifications@github.com> wrote:

> In recipes/snappy/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/193#discussion_r57983056>
> :
>
> > +{% set version = "1.1.3" %}
> > +
> > +package:
> > +  name: snappy
> > +  version: {{ version }}
> > +
> > +source:
> > +  git_url: https://github.com/google/snappy.git
>
>
> > +  git_tag: {{ version }}
> > +
> > +requirements:
> > +  build:
> > +    - msinttypes  # [win]
> > +    - libtool # [unix]
> > +    - automake # [unix]
> > +
>
> The test is executed by make check in build.sh. What other tests do you
> recommend?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/193/files/67d524be2767352d3295d6f110c67f718cf6ebe6#r57983056>
>